packages/components/src/components/wallet-icon/wallet-icon.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import classNames from 'classnames';
 import Icon from '../icon';
+import TradingPlatformIcon from '../trading-platform-icon';
+import { IconProps } from '../trading-platform-icon/trading-platform-icon';
 import './wallet-icon.scss';
 
 type TWalletIconSizes = keyof typeof sizes['square' | 'box'];
 
 type TWalletIconProps = {
     gradient_class?: string;
-    icon: string;
+    icon: string | IconProps<string>['icon'];
```

Comment:
@aizad-deriv don't understand the sense of adding `IconProps<string>['icon']` if we still have `string` as a type